### PR TITLE
Seeding for availability

### PR DESCRIPTION
**Pull request by slava000**

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const { Sequelize, DataTypes } = require('sequelize');
+const schema = require('./seed.js');
+const sequelize = new Sequelize('availability', 'root', 'sqlcraft', {
```
Comment (manedurphy) [inline database/index.js]: Store your database credentials in environment variables

**Review by manedurphy — COMMENTED** [review]
@slava000 based on my last conversation with Leslie, our seed script should be entirely separate from the methods we use to initialize a connection to our database to ensure our seed script only runs once when we deploy.

File: database/index.js (hunk 1 above)
Comment (slava000) [reply database/index.js]: thanks @manedurphy!! I just pushed changes to address your comments. Could you please review when you get a chance?

File: database/index.js [hunk 2]
```diff
@@ -1,7 +1,10 @@
 const { Sequelize, DataTypes } = require('sequelize');
 const schema = require('./seed.js');
-const sequelize = new Sequelize('availability', 'root', 'sqlcraft', {
-  host: 'localhost',
+const dotenv = require('dotenv').config({ path: '../vars'});
+const sequelize = new Sequelize('availability', {
```
Comment (manedurphy) [inline database/index.js]: Nice!

File: database/seed.js [hunk 3]
```diff
@@ -1,9 +1,5 @@
 const { Sequelize, DataTypes } = require('sequelize');
 const schema = require('./index.js');
-const sequelize = new Sequelize('availability', 'root', 'sqlcraft', {
-  host: 'localhost',
-  dialect: 'mysql'
-})
 
 //Verify connection
 const auth = async ()=> {
```
Comment (manedurphy) [inline database/seed.js]: I noticed this same function was in your index.js file. It doesn't look like it's being invoked in this file so try deleting it and then see if your seeding script still works.

**Review by manedurphy — CHANGES_REQUESTED** [review]
Nice change on the db credentials. I noticed some repeated code that can be taken out. I've left a comment on the exact line.

**Pull request merged**